src/behavior-considered-undefined.md

```diff
@@ -49,10 +49,11 @@ Please read the [Rustonomicon] before writing unsafe code.
   All this also applies when values of these
   types are passed in a (nested) field of a compound type, but not behind
   pointer indirections.
-* Mutating immutable bytes. All bytes inside a [`const`] item are immutable.
+* Mutating immutable bytes. All bytes inside a [`const`] item are immutable, including implicitly [const-promoted] expressions.
```

Comment:
```suggestion
* Mutating immutable bytes. All bytes inside a [`const`] item or within an implicitly [const-promoted] expression are immutable.
```

Something like this might be clearer.  The "implicitly const-promoted expressions" are not in the set of things "inside const items", so saying "including" here is not quite right.

Reply:
Const promotion can be seen as creating const items, and then "including" makes sense... but maybe that kind of thinking is too implementation-focused. I applied your suggestions.